src/daemon/manager.py

```
@@ -151,3 +160,182 @@ def get_all_jobs_status() -> Dict[str, Any]:
+                            "source": "mongodb_fallback"
+                        }
+                    
+                    mongo_client.close()
+                else:
+                    mongo_client.close()
+            except Exception as mongo_e:
+                pass
+        
+        # Second fallback: Try to read from local state files
+        try:
+            from ..core.state import BlazeState
```

Ruff (0.12.2)

`..core.state.BlazeState` imported but unused

Remove unused import: `..core.state.BlazeState`

(F401)